pkg/kube/apisix/apis/config/v2beta2/types.go

```diff
@@ -140,6 +141,14 @@ type ApisixRouteAuthenticationKeyAuth struct {
 	Header string `json:"header,omitempty" yaml:"header,omitempty"`
 }
 
+// ApisixRouteAuthenticationJwtAuth is the jwtAuth-related
+// configuration in ApisixRouteAuthentication.
+type ApisixRouteAuthenticationJwtAuth struct {
```

Comment:
we are going to add new feature to v2beta2? @tao12345666333 is it by design?

Reply:
no, we don't need add any feature to v2beta2.  I missed.
 @AlinsRan please remove `v2beta2` related code. we only need add feature to `v2beta3` and `v2`

Reply:
OK.